undetected_chromedriver/__init__.py

```diff
@@ -637,8 +748,7 @@ def quit(self):
 
     def __del__(self):
         try:
-            super().quit()
-            # self.service.process.kill()
+            self.service.process.kill()
```

Comment:
what's the motivation for this?

regular chromedriver shuts down at this point.